app/services/service_listeners/publishing_api_pusher.rb

```diff
@@ -35,15 +35,30 @@ def push(event:, options: {})
         api.discard_draft_async(edition)
       end
 
+      handle_corporate_information_pages(event)
       handle_html_attachments(event)
+      handle_publications(event)
     end
 
   private
 
+    def handle_corporate_information_pages(event)
```

Comment:
It doesn't republish the corporate information pages or publications, it republished the about pages of the organisation's related to the corporate information pages or publications. Since that behaviour is contained within those two workers all we know at this stage is that we need to invoke the worker.